services/core-web/common/utils/helpers.js

```diff
@@ -464,3 +464,29 @@ export const isPitsQuarriesAdjustmentFeeValid = (proposed = 0, adjusted = 0) =>
   // Anything above 170,000 is valid as the applicant already paid the max fee.
   return isFeeValid;
 };
+
+export const determineInspectionFeeStatus = (
```

Comment:
This should be called "exemption fee", not "inspection fee"? I think we reference it as inspection fee in other places too.

Reply:
updated the name to be:  determineExemptionFeeStatus